src/ServiceManagement/RemoteApp/Commands.RemoteApp.Tests/Collection/RemoteAppCollection.cs

```diff
@@ -0,0 +1,323 @@
+            {
+                Assert.Fail(
+                    String.Format("Get-AzureRemoteAppCollection returned the following error {0}.",
+                        mockCmdlet.runTime().ErrorStream[0].Exception.Message
+                    )
+                );
+            }
+
+            List<Collection> collections = MockObject.ConvertList<Collection>(mockCmdlet.runTime().OutputPipeline);
+            Assert.IsNotNull(collections);
+
+            Assert.IsTrue(collections.Count == countOfExpectedCollections,
+                String.Format("The expected number of collections returned {0} does not match the actual {1}.",
+                    countOfExpectedCollections,
+                    collections.Count
+                )
+            );
+
+            Assert.IsTrue(MockObject.HasExpectedResults<Collection>(collections, MockObject.ContainsExpectedCollection),
+                "The actual result does not match the expected."
+            );
+
+            Log("The test for Get-AzureRemoteAppCollection with {0} collections completed successfully", countOfExpectedCollections);
+        }
+
+
+        [TestMethod]
+        public void GetCollectionsByName()
+        {
+            int countOfExpectedCollections = 1;
+            GetAzureRemoteAppCollection mockCmdlet = SetUpTestCommon<GetAzureRemoteAppCollection>();
+
+            // Required parameters for this test
+            mockCmdlet.CollectionName = collectionName;
+
+            // Setup the environment for testing this cmdlet
+            MockObject.SetUpDefaultRemoteAppCollectionByName(remoteAppManagementClientMock, collectionName);
+            mockCmdlet.ResetPipelines();
+
+            Log("Calling Get-AzureRemoteAppCollection to get this collection {0}.", mockCmdlet.CollectionName);
+
+            mockCmdlet.ExecuteCmdlet();
+
+            if (mockCmdlet.runTime().ErrorStream.Count != 0)
+            {
+                Assert.Fail(
+                    String.Format("Get-AzureRemoteAppUser returned the following error {0}.",
+                        mockCmdlet.runTime().ErrorStream[0].Exception.Message
+                    )
+                );
+            }
+
+            List<Collection> collections = MockObject.ConvertList<Collection>(mockCmdlet.runTime().OutputPipeline);
+            Assert.IsNotNull(collections);
+
+            Assert.IsTrue(collections.Count == countOfExpectedCollections,
+               String.Format("The expected number of collections returned {0} does not match the actual {1}.",
+                   countOfExpectedCollections,
+                   collections.Count
+               )
+           );
+
+            Assert.IsTrue(MockObject.HasExpectedResults<Collection>(collections, MockObject.ContainsExpectedCollection),
+               "The actual result does not match the expected."
+           );
+
+           Log("The test for Get-AzureRemoteAppCollection with {0} collections completed successfully", countOfExpectedCollections);
+        }
+
+
```

Comment:
Nope. The previous method ends on line 112. Your next method should start at line 114 and not line 115.
There is an extra line which is not required. You can use the auto-format feature in Visual studio (Edit-> Advanced-> FormatDocument).